Documentation/requirements.md

```diff
@@ -0,0 +1,35 @@
+
+Depending on the [platform](Platforms.md) that you are targeting, MonoGame has different sets of requirements.
+
+## For desktop platforms
+
+MonoGame requires a .NET Core SDK (3.1 or up) installation.
+
+You can either install it [independently](https://dotnet.microsoft.com/download/dotnet-core), or by selecting the .NET Core payload when installing Visual Studio 2019 (version 15.4 and up required).
```

Comment:
[nit] I believe the terminology VS uses is "workload", not "payload".

I've written a lot of docs for people to set up VS 😛